Jenkinsfile

```diff
@@ -0,0 +1,142 @@
+/* groovylint-disable DuplicateStringLiteral */
+/* groovylint-disable-next-line CompileStatic */
+pipeline {
+    agent any
+    options {
+        buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '5', numToKeepStr: '5')
+        disableConcurrentBuilds()
+    }
+    stages {
+        stage('ci') {
+            agent {
+                docker {
+                    image 'node:18-alpine'
+                    args '-u root'
+                }
+            }
+            environment {
+                NODE_ENV = 'test'
+                NODE_PORT = 3131
+                ARANGO_DB_HOST_DOMAIN = 'arangotest.digiteched.com'
+                ARANGO_DB_HOST_SCHEME = 'https'
+                ARANGO_DB_HOST_PORT = 443
+                ARANGO_DB_ROOT_PASSWORD = credentials('arango-test-root-password')
+                ARANGO_DB_USER = 'dbuser'
+                ARANGO_DB_USER_PASSWORD = credentials('arango-test-db-user-password')
+                ARANGO_DB_NAME = 'staging'
+                AUTH0_ISSUER_URL = 'https://foo.auth0.com/'
+                AUTH0_AUDIENCE = 'https://bar.mydomain.ca'
+                BASE_DIGITAL_ASSET_URL = 'https://www.myaudio.com:5555/media?id='
+                GLOBAL_PREFIX = 'api'
+                SHOULD_ENABLE_LEGACY_GAMES_ENDPOINT = 'false'
+            }
+            when {
+                branch 'PR-*'
+            }
+            steps {
+                configFileProvider([configFile(fileId:'42feff14-78da-45fc-a8ee-5f98213a313f',  \
+            targetLocation: 'apps/coscrad-frontend/src/auth_config.json')]) {
+                    echo 'PR opened or updated...'
+                    echo "NODE ENV: ${NODE_ENV}"
+                    echo 'Installing dependencies'
+                    sh 'npm ci --legacy-peer-deps'
+
+                    echo 'Running lint on all COSCRAD projects'
+                    sh 'npm run lint:coscrad'
+
+                /**
+                * Note that the sample content config is actually valid for
+                * our staging build.
+                **/
+                    echo 'copying sample content config for test build'
+                /* groovylint-disable-next-line LineLength */
+                    sh 'cp apps/coscrad-frontend/src/configurable-front-matter/data/content.config.SAMPLE.ts apps/coscrad-frontend/src/configurable-front-matter/data/content.config.ts'
```

Comment:
Be nice to use multi-line strings, but I'm not too worried about learning Groovy syntax right now.